src/whispercpp/context.cc

```diff
@@ -342,6 +342,9 @@ void ExportContextApi(py::module &m) {
       .def("lang_detect", &Context::lang_detect, "offset_ms"_a, "threads"_a = 1)
       .def("get_logits", &Context::get_logits, "segment"_a)
       .def("token_to_str", &Context::token_to_str, "token_id"_a)
+      .def("token_to_bytes", [](Context & context, whisper_token token_id) {
+          return py::bytes(context.token_to_str(token_id));
+      }, "token_id"_a)
```

Comment:
Should we just move this to python completely with `py::return_value_policy::move`?